root/etc/s6-overlay/s6-rc.d/svc-mod-universal-cloudflared/run

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/with-contenv bash
 
 if [[ $(uname -m) = "armv7l" ]]; then
-  echo "**** The universal cloudflared mod no longer supports arm32v7/armhf per https://info.linuxserver.io/issues/2023-07-01-armhf/ ****"
-  sleep infinity
+    echo "**** The universal cloudflared mod no longer supports arm32v7/armhf per https://info.linuxserver.io/issues/2023-07-01-armhf/ ****"
+    sleep infinity
 fi
```

Comment:
Can probably remove the armhf check at this point, if anyone is still running one of those images somewhere god help them.

Reply:
Done